flasgger/utils.py

```diff
@@ -155,18 +155,19 @@ def get_specs(rules, ignore_verbs, optional_fields, sanitizer, doc_dir=None):
 
                 swagged = True
 
-            if doc_dir:
-                if view_class:
-                    file_path = os.path.join(
-                        doc_dir, endpoint.__name__, method.__name__ + '.yml')
-                else:
-                    file_path = os.path.join(
-                        doc_dir, endpoint.__name__ + '.yml')
-                if os.path.isfile(file_path):
-                    func = method.__func__ \
-                        if hasattr(method, '__func__') else method
-                    setattr(func, 'swag_type', 'yml')
-                    setattr(func, 'swag_path', file_path)
+            if doc_dir and isinstance(doc_dir, Iterable):
+                for _dir in doc_dir:
+                    if view_class:
+                        file_path = os.path.join(
+                            _dir, endpoint.__name__, method.__name__ + '.yml')
+                    else:
+                        file_path = os.path.join(
+                            _dir, endpoint.__name__ + '.yml')
+                    if os.path.isfile(file_path):
+                        func = method.__func__ \
+                            if hasattr(method, '__func__') else method
+                        setattr(func, 'swag_type', 'yml')
+                        setattr(func, 'swag_path', file_path)
```

Comment:
What would happen if there are multiple doc_dir? Will ```setattr(func, 'swag_type', 'yml')``` override previous definition of func? It would be better if there is a unittest or test case to demonstrate.